src/Control/Session.php

```diff
@@ -6,6 +6,16 @@
 use SilverStripe\Core\Config\Configurable;
 use SilverStripe\Dev\Deprecation;
 
+use function headers_sent;
```

Comment:
I don't think it's common to specifically `use` native methods and constants, is there any specific/functional reason for adding these here?

Reply:
Hello @michalkleiner,

PHP will look for a `header_sent` function in the current namespace first and then in the global namespace. Importing native functions and constants can speed up an application up to 5%.
kind regards

Reply:
Appreciate that @pine3ree, I could've been more specific and said that it's not a common approach in Silverstripe, so I wouldn't want to introduce it in an unrelated enhancement.

If we decided to add `use` statements for all native functions used in a class, we would probably do it bulk across the whole framework through some automation/code analysis tools.